MI-Xib_cell/Controllers/ViewController.swift

```diff
@@ -43,16 +41,13 @@ extension ViewController: UITableViewDelegate, UITableViewDataSource {
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         guard let cell = tableView.dequeueReusableCell(withIdentifier: CustomTableViewCell.reuseId, for: indexPath) as? CustomTableViewCell else {return .init()}
         
-        cell.configure(labelList: labelData)
+        let indexOfRow = labelData[indexPath.row]
+        cell.configure(labelList: labelData, indexOfRow: indexOfRow)
```

Comment:
Why you send labelData?

Reply:
I send labelData to get label text data from ViewController to cell file and execute the configure function.

Reply:
what is difference between indexOfRow?